services/supervisor/src/proto/service.proto

```diff
@@ -74,6 +74,8 @@ message ConfigPropose {
   uint64 actual_from = 1;
   // New configuration proposition.
   repeated ConfigChange changes = 2;
+  // Configuration proposal number to avoid conflicting proposals.
+  uint64 configuration_number = 3;
```

Comment:
Just to clarify. `uint32` is not enough ?

Reply:
Well, *in theory* it's possible to have a new configuration on each height, and height is `uint64`.
I thought that since there is some correlation between height and configuration number, it's better to have the same type.